.envrc

```diff
@@ -14,7 +14,8 @@ PYTHON=$(type -p python3.4 || type -p python3 || type -p python)
 
 layout python $PYTHON
 
-export DATADIR="data"
+export DATADIR="data/2018-07-10"
```

Comment:
I'd perhaps avoid committing this change, although the default value isn't much better.

Reply:
I think I fixed this. But I never know if my interactive rebase parties actually work...

Reply:
nope - doesn't seem to have done it. Will try again later.